frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1181,6 +1463,12 @@ export const surveyLogic = kea<surveyLogicType>([
                 return !!enabledFlags[FEATURE_FLAGS.SURVEYS_PARTIAL_RESPONSES]
             },
         ],
+        isNewQuestionVizEnabled: [
```

Comment:
new feature flag I'll use to switch between new viz vs old viz, in case something goes wrong we can easily revert it.

for now this does nothing, but it'll be used once we actually add the new components.

the new query is not executed at all if this is false